### PR TITLE
release(v0.12.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.12.0 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,382 @@
...
+
+Sysctl Kernel Params configuration was completely rewritten to be based on controllers and resources,
+which makes it possible to apply `.machine.sysctls` in immediate mode (without a reboot).
+`talosctl get kernelparams` returns merged list of KSPP, Kubernetes and user defined params along with
+the default values overwritten by Talos.
+
+
+### Equinix Metal
+
+Added support for Equinix Metal IPs for the Talos virtual (shared) IP (option `equnixMetal` under `vip` in the machine configuration).
```
Comment (andrewrynhard) [inline CHANGELOG.md:45]: `s/equnixMetal/equinixMetal`

Comment (smira) [reply CHANGELOG.md:45]: :facepalm:

Comment (smira) [reply CHANGELOG.md:45]: :man_facepalming:

**Pull request merged**